content/sklearn/concepts/support-vector-machines/support-vector-machines.md

```diff
@@ -0,0 +1,74 @@
+---
+Title: 'Support Vector Machines'
+Description: 'Support Vector Machines are a supervised learning algorithm used to classify data by finding a separation line between categories.'
+Subjects:
+  - 'Data Science'
+  - 'Machine Learning'
+Tags:
+  - 'Supervised Learning'
+  - 'Sci-kit Learn'
+CatalogContent:
+  - 'getting-started-with-python-for-data-science'
+  - 'paths/data-science'
+---
+
+**Support Vector Machines**(SVMs) are a supervised learning algorithm excelling at classification tasks. They work by finding the optimal hyperplane that maximizes the margin between different classes in your data. The margin is the distance between the hyperplane and the closest data points from each class, called support vectors. SVMs are particularly effective for high-dimensional datasets, and handle complex data while promoting good generalization and reducing the risk of overfitting.
+
+## Example
```

Comment:
Move the example section after the `Syntax` subheading. The example section should contain a code block and its output.